analyses/pypi/generatepylist.go

```diff
@@ -0,0 +1,16 @@
+package pypi
+
+// CodeForPylist is converted to generate_pylist.py
+var CodeForPylist = []byte(
```

Comment:
I would still prefer `go generate` approach though it seems to add additional build step. Additional step is not going to affect cli users any way.

Reply:
I see what you’re saying but I don't feel adding an extra build step specifically for Pypi is needed unless it is required for every ecosystem individually.